### PR TITLE
Move SSO reference into conceptual temporarily

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
This is a temporary solution: We need this beta API in the documentation but it must not appear in the Intellisense. So moving it to the SSO conceptual topic. It will be pulled from the d.ts in a PR to the DefinitelyTyped repo.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/troubleshoot-sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/troubleshoot-sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809052246078101-358%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2BQsS2kM8F%2BTNg%2B1QLOeY5JUjaVF0VIHMODavsrvgli4%3D&st=2018-09-05T22%3A42%3A20Z&se=2018-10-06T22%3A47%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/troubleshoot-sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/troubleshoot-sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809052330369729-358%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Rp7IEnTaBYwYdENCgZvWJasdxkxkECC0SrEty9DpLvU%3D&st=2018-09-05T23%3A26%3A49Z&se=2018-10-06T23%3A31%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/troubleshoot-sso-in-office-add-ins.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ ms.date: 12/08/2017
 
 This article provides some guidance about how to troubleshoot problems with single sign-on (SSO) in Office Add-ins, and how to make your SSO-enabled add-in robustly handle special conditions or errors.
 
+> [!NOTE]
+> The Single Sign-on API is currently supported in preview for Word, Excel, Outlook, and PowerPoint. For more information about where the Single Sign-on API is currently supported, see [IdentityAPI requirement sets](https://dev.office.com/reference/add-ins/requirement-sets/identity-api-requirement-sets).
```
Comment (AlexJerabek) [inline docs/develop/troubleshoot-sso-in-office-add-ins.md]: Please update the link to docs.microsoft.com.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/troubleshoot-sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/troubleshoot-sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809052340000747-358%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=pQOgV6vDVA3oScn75nfGdoCokPIiqpMyyj1rmrJYwE8%3D&st=2018-09-05T23%3A36%3A09Z&se=2018-10-06T23%3A41%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/sso-in-office-add-ins.md [hunk 2]
```diff
@@ -203,4 +203,57 @@ The following is a typical decoded payload of an access token. For information a
 
 ## Using SSO with and Outlook add-in
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/sso-in-office-add-ins.md]: Please update to "an Outlook"

File: docs/develop/sso-in-office-add-ins.md [hunk 3]
```diff
@@ -203,4 +203,57 @@ The following is a typical decoded payload of an access token. For information a
...
 
-There are some small, but important differences in using SSO in and Outlook add-in from using it in as Excel, PowerPoint, or Word add-in. Be sure to read [Authenticate a user with a single sign-on token in an Outlook add-in](https://docs.microsoft.com/outlook/add-ins/authenticate-a-user-with-an-sso-token) and [Scenario: Implement single sign-on to your service in an Outlook add-in](https://docs.microsoft.com/outlook/add-ins/implement-sso-in-outlook-add-in).
\ No newline at end of file
+There are some small, but important differences in using SSO in and Outlook add-in from using it in as Excel, PowerPoint, or Word add-in. Be sure to read [Authenticate a user with a single sign-on token in an Outlook add-in](https://docs.microsoft.com/outlook/add-ins/authenticate-a-user-with-an-sso-token) and [Scenario: Implement single sign-on to your service in an Outlook add-in](https://docs.microsoft.com/outlook/add-ins/implement-sso-in-outlook-add-in).
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/sso-in-office-add-ins.md]: should be "SSO in an Outlook"

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/authorize-to-microsoft-graph.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/authorize-to-microsoft-graph.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-aspnet.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/create-sso-office-add-ins-aspnet.md) | :white_check_mark:Succeeded | |
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/troubleshoot-sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/getAccessTokenAsync-ref-into-conceptual/docs/develop/troubleshoot-sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201809062237579228-358%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hvxftldKZrYYb%2FyX3VIZZXjJR3Vq%2FhFRi%2B70bkdseZc%3D&st=2018-09-06T22%3A34%3A06Z&se=2018-10-07T22%3A39%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**